requirements.txt

```diff
@@ -1,14 +1,7 @@
 bcrypt==3.2.0
```

Comment:
Can we retain the other requirements that we will need in the future (like GitPython) and just update their versions? Rather than just removing them right away.